xmr-btc/src/bob.rs

```diff
@@ -620,6 +558,7 @@ impl State3 {
             tx_lock: self.tx_lock,
             tx_cancel_sig_a: self.tx_cancel_sig_a,
             tx_refund_encsig: self.tx_refund_encsig,
+            monero_rescan_block_height,
```

Comment:
```suggestion
            lock_transaction_height,
```

Instead of naming it after *how* we want to use it, let's name it after *what* it is.

Reply:
I would opt for doing this in the follow up. At the moment it is not precisely the `lock_transaction_height`, but a height that we record before we see the lock tx that is recorded for the sole purpose of using it for the wallet creation. I would make the name explicit, otherwise it could be confusing because one might think the height is relevant for watching the transactions. Once we implement extracting the exact height from the transfer most of what is currently there would be removed and the remaining parts can be renamed.